examples/bristlecone_heatmap_example.py

```diff
@@ -1,16 +1,57 @@
-"""The script that produces bristlecone_heatmap_example.png."""
+"""Example heatmaps from the cirq.vis.heatmap package."""
 
```

Comment:
Let's clean this example up a bit: 
- the name is now not only bristelcone_heatmap 
- bristlecone could be renamed to single_qubit 
- instead of edges, let's use the word "qubit_pairs"

Reply:
done